astrodendro/io/util.py

```diff
@@ -140,13 +140,19 @@ def _fast_reader(index_map, data):
 
     # ndimage ignores 0 and -1, but we want index 0
     object_slices = ndimage.find_objects(index_map+1)
+
+    # find_objects returns a tuple that includes many None values that we
+    # need to get rid of.
+    object_slices = [x for x in object_slices if x is not None]
+
```

Comment:
@ChrisBeaumont - I'm really confused as to how the fast_reader ever worked before! Even going back to scipy 0.11 it looks like this is needed. Weird!